website/docs/r/vapp_vm.html.markdown

```diff
@@ -377,8 +367,7 @@ This field works as a flag and triggers force customization when `true` during a
 is needed after VM configuration (e.g. NIC change, customization options change, etc.) and then set back to `false`.
 **Note.** It will not have effect when `power_on` field is set to `false`. See [example workflow below](#example-forced-customization-workflow).
 * `enabled` (Optional; *v2.7+*) `true` will enable guest customization which may occur on first boot or if the `force` flag is used.
-This option should be selected for **Power on and Force re-customization to work**. For backwards compatibility it is
-enabled by default when deprecated field `initscript` is used.
+This option should be selected for **Power on and Force re-customization to work**.
```

Comment:
We have deprecated `power_on` in the previous PR https://github.com/vmware/terraform-provider-vcd/pull/558
Won't that cause issues with the above?
CC: @Didainius

Reply:
VM have own power on handling, no issue here. Issue could be if power_on would be used together on VM and vApp.